### ISSUE TITLE
revise responsive toggling methodology

### ISSUE THREAD

**Issue by schasoli** [drupalprojects/kalatheme#146]
Hello,
with <a href="https://www.drupal.org/project/breakpoint_panels">breakpoint_panels</a> panels can be not rendert do to breakpoints (not only hidden). It might be worth adding this into Using-Kalatheme-and-Best-Practices.
I am just not sure, if this is wanted.
cheers
Oli

**Comment by pirog** [comment]
yes this should def be considered the preferred way to do this sort of thing. adding it to the docs would be a great idea.

**Comment by thiagodemellobueno** [comment]
For accessibility we may want to have it be an 'either or' (sometimes you want to keep important content but hide from visual browsers).

**Comment by andrewmallis** [comment]
as part of this, [Responsive Toggling](https://github.com/drupalprojects/kalatheme/wiki/Responsive-Toggling) wiki page should be updated.
